cmd/istioctl/main.go

```diff
@@ -271,7 +300,7 @@ func init() {
 	rootCmd.AddCommand(putCmd)
 	rootCmd.AddCommand(getCmd)
 	rootCmd.AddCommand(deleteCmd)
-	rootCmd.AddCommand(injectCmd)
+	// rootCmd.AddCommand(injectCmd)
```

Comment:
We can ship istioctl with all platform specific sub commands. If the user tries to do kube-inject against a mesos cluster, the command would ultimately fail.